README.md

```diff
@@ -1,75 +1,44 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Capstone Provect/visit RWANDA
 
-> One paragraph statement about the project.
+> Visit Rwanda is a project that provide enough info or all you want to know about VISIT Rwanda program, this project is builded in html css and javascript, some contents of the page generate dynamical when page reload.
 
-![screenshot](./app_screenshot.png)
+![screenshot](./app_screenshot.PNG)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Javascript
+- HTML
+- CSS
 
```

Comment:
- [ ] Your Pull Request does not include a video presentation showcasing your project. Kindly endeavor to record a maximum of 5 minutes video presenting the features of the project. The video should:
- contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- contain a demo of the project, showing the pages in different screen sizes.
- contain a highlight about a piece of the code the student finds interesting or is proud of making.